R/driver-spark.R

```diff
@@ -12,6 +12,9 @@ setClass("Spark SQL", contains = "OdbcConnection")
 #' @usage NULL
 setMethod("odbcConnectionSchemas", "Spark SQL",
   function(conn, catalog_name) {
```

Comment:
```suggestion
  function(conn, catalog_name = NULL) {
```

To make it clear that the argument is optional?